policyengine_us/parameters/gov/states/mn/tax/income/credits/cdcc/phaseout_threshold.yaml

```diff
@@ -2,13 +2,20 @@ description: Minnesota has a child/dependent care credit which begins to be phas
 values:
   2021-01-01: 53_630
   2022-01-01: 55_300
+  2023-01-01: 59_210
 
 metadata:
   period: year
   unit: currency-USD
   label: Minnesota CDCC phaseout threshold
   reference:
+  - title: Minnesota Statutes 290.067 DEPENDENT CARE CREDIT. Subd. 1.Amount of credit. (h)
```

Comment:
I think we should add uprating here:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/bb9b0495-fc7c-45f4-8306-a75b2cc9e737)